docs/2.8.0/docs/daml-finance/tutorials/lifecycling/fixed-rate-bond.rst

```diff
@@ -86,6 +86,20 @@ first coupon:
   :start-after: -- CREATE_CLOCK_UPDATE_EVENT_BEGIN
   :end-before: -- CREATE_CLOCK_UPDATE_EVENT_END
 
+Note that it is the bank that actively creates a
```

Comment:
I wonder if this paragraph should be part of a `Time management` sub-section at the end of the tutorial, in order not to break the current flow too much. I'll leave it up to you to decide

Reply:
Good point. I think I will keep it where it is for now (since it refers to different parts of the tutorial code), but will consider moving it later on if it looks too clunky (especially if we want to add more content on this topic).